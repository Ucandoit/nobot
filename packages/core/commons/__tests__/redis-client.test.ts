import redisClient from '../src/redis-client';

beforeAll(() => {
  redisClient.start();
  redisClient.set('test1', '1');
  redisClient.set('test2', '2');
});

afterAll(() => {
  redisClient.disconnect();
});

test('redis get', async () => {
  const val = await redisClient.get('test1');
  expect(val).toMatchSnapshot();
});
