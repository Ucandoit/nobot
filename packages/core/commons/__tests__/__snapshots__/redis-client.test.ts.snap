// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redis get 1`] = `"1"`;
